centinel/unit_test/test_dnslib.py

```diff
@@ -0,0 +1,176 @@
+import pytest
+import re
+import os
+from centinel.primitives import dnslib
+
+
+class TestDnslib:
+
+    @pytest.fixture(scope='class')
+    def ipregex(self):
+        return "^((((2(([0-4]\d)|(5[0-5]))\.)|(1\d\d\.))|([1-9]\d\.))|(\d\.)){3}((((2(([0-4]\d)|(5[0-5]))$)|(1\d\d$))|([1-9]\d$))|(\d$))"
+
+    @pytest.fixture(scope='class')
+    def invalid_domains(self):
+        cwd = os.getcwd()
+        invalid_domains = [];
+        with open(os.path.join(cwd,'centinel/data/invalid_hosts.txt')) as testfile:
```

Comment:
This path is from one directory above. Why is the datafile for the unit test in another directory? Why not `cwd/data`?